jdaviz/components/viewer_data_select.vue

```diff
@@ -142,7 +142,7 @@ module.exports = {
           return false
         } else if (this.$props.viewer.reference === 'image-viewer' && item.type !== 'image') {
           return false
-        } else if (item.meta.mosviz_row !== undefined) {
+        } else if (item.meta.mosviz_row !== null) {
```

Comment:
I think because of the logic that sets meta, we also need to catch the undefined case (which will happen if `mosviz_row` does not appear as a key in the `meta` dictionary):

https://github.com/spacetelescope/jdaviz/blob/8b136cc385c4569a50cc2ff84c90ebf184f52a90/jdaviz/app.py#L1512

Reply:
Yep, I realized that after I pushed. Implemented in 
https://github.com/spacetelescope/jdaviz/pull/1798/commits/df1644f79ad5d7af83df3cf895c7d2606b22379e

Reply:
Fixed in https://github.com/spacetelescope/jdaviz/pull/1798/commits/eaca4f6cc279682c5bb2e426e62c219e9e91ab95